docs/bills-matching.md

```diff
@@ -0,0 +1,139 @@
+# Bills/transactions matching algorithm
+
+The matching algorithm is in charge of finding matchings between
+`io.cozy.bills` and `io.cozy.bank.operations` documents. This document explains
+how the matching algorithm works.
+
+## The Linker
+
+The algorithm is held by the [`Linker`](), and more precisely its `linkBillsToOperations` method. This method takes 3 parameters:
+
+* An array of `io.cozy.bills` objects
+* An array of `io.cozy.bank.operations` objects
+* An options object
+
+The options object can contain the following properties:
+
+* `minAmountDelta`: defaults to `0.001`
+* `maxAmountDelta`: defaults to `0.001`
+* `pastWindow`: defaults to 15 (days)
+* `futureWindow`: defaults to 29 (days)
```

Comment:
Is it properly documented with jsdoc ? I think this documentation could live in the Linker file. I wonder how could we "embed"/"mix" something generated from JSDoc and a more detailed documentation like this one.

Reply:
https://hacks.mozilla.org/2017/07/introducing-sphinx-js-a-better-way-to-document-large-javascript-projects/

Reply:
This is now properly documented with jsdoc